spec/services/participants/ecf_spec.rb

```diff
@@ -1,63 +0,0 @@
-# frozen_string_literal: true
```

Comment:
I think we can keep this spec as well as the one below as we're still using those in the remaining action methods, we can delete those in the end?